lldb/test/Shell/Driver/TestSingleQuote.test

```diff
@@ -1,3 +1,4 @@
+# XFAIL: target=x86_64-{{.*}}-windows{{.*}}
 # Make sure lldb can handle filenames with single quotes in them.
 # RUN: %clang_host %p/Inputs/hello.c -g -o "%t-'pat"
```

Comment:
And this looks like you may have a different shell (one that handles quotes differently) from the other windows devs/bots. Maybe we can figure out an invocation that works with both? @DavidSpickett

Reply:
I'm pretty sure we run in cmd.exe. This is how we start the buildbot: https://linaro.atlassian.net/wiki/spaces/TCWGPUB/pages/25310167965/How+to+setup+LLVM+buildbots+for+Windows+on+Arm#Buildbot-startup-script

Reply:
The same as for TestConvenienceVariables.test:

```
PASS: lldb-shell :: Driver/TestSingleQuote.test (75 of 543)
Exit Code: 0

Command Output (stdout):
--
# RUN: at line 2
c:\build-lldb-native\bin\clang.exe --target=specify-a-target-or-use-a-_host-substitution --target=x86_64-pc-windows-msvc -fmodules-cache-path=C:/build-lldb-native/lldb-test-build.noindex/module-cache-clang\lldb-shell C:\lldb\test\Shell\Driver/Inputs/hello.c -g -o "C:\build-lldb-native\tools\lldb\test\Shell\Driver\Output\TestSingleQuote.test.tmp-'pat"
# executed command: 'c:\build-lldb-native\bin\clang.exe' --target=specify-a-target-or-use-a-_host-substitution --target=x86_64-pc-windows-msvc '-fmodules-cache-path=C:/build-lldb-native/lldb-test-build.noindex/module-cache-clang\lldb-shell' 'C:\lldb\test\Shell\Driver/Inputs/hello.c' -g -o 'C:\build-lldb-native\tools\lldb\test\Shell\Driver\Output\TestSingleQuote.test.tmp-'"'"'pat'
# .---command stderr------------
# | clang: warning: argument unused during compilation: '-fmodules-cache-path=C:/build-lldb-native/lldb-test-build.noindex/module-cache-clang\lldb-shell' [-Wunused-command-line-argument]
# `-----------------------------
# RUN: at line 3
c:\build-lldb-native\bin\lldb.exe --no-lldbinit -S C:/build-lldb-native/tools/lldb\test\Shell\lit-lldb-init-quiet -s C:\lldb\test\Shell\Driver\TestSingleQuote.test "C:\build-lldb-native\tools\lldb\test\Shell\Driver\Output\TestSingleQuote.test.tmp-'pat" | c:\build-lldb-native\bin\filecheck.exe C:\lldb\test\Shell\Driver\TestSingleQuote.test
# executed command: 'c:\build-lldb-native\bin\lldb.exe' --no-lldbinit -S 'C:/build-lldb-native/tools/lldb\test\Shell\lit-lldb-init-quiet' -s 'C:\lldb\test\Shell\Driver\TestSingleQuote.test' 'C:\build-lldb-native\tools\lldb\test\Shell\Driver\Output\TestSingleQuote.test.tmp-'"'"'pat'
# note: command had no output on stdout or stderr
# executed command: 'c:\build-lldb-native\bin\filecheck.exe' 'C:\lldb\test\Shell\Driver\TestSingleQuote.test'
# note: command had no output on stdout or stderr

--

```